ceda_status_validator/validate.py

```diff
@@ -0,0 +1,11 @@
+from model import StatusPage
+from pydantic import ValidationError
+
+with open("status.json", "r") as status_file:
+    status_page = status_file.read()
```

Comment:
Should be:

`status_page = json.load(status_file)`

Reply:
Can't use `json.load` because Pydantic's `model_validate_json` wants a string not an object

Reply:
Seems a bit weird, but it does seem to be pydantic's recommended way of doing it: https://docs.pydantic.dev/latest/examples/files/